enterprise/server/index.md

```diff
@@ -109,16 +115,19 @@ For other versions of Tomcat on Windows, check the Tomcat documentation for that
 
 If you're following the instructions for [Starting Jetty](https://www.eclipse.org/jetty/documentation/current/startup.html) for Jetty 9.4.5, the path to the configuration file can simply be supplied as a command option:
 
-    java -D"ephox.config.file=C:\config\file\location\application.conf" -jar C:\jetty\install\directory\start.jar
-
+```
+java -D"ephox.config.file=C:\config\file\location\application.conf" -jar C:\jetty\install\directory\start.jar
+```
 
 For other versions of Jetty on Windows, check the Jetty documentation for that version.
 
 ###### As a Windows service
 
 Follow the instructions in [Startup via Windows Service](https://www.eclipse.org/jetty/documentation/current/startup-windows-service.html) for Jetty 9.4.5. Remember to append the following snippet to the line beginning with `set PR_JVMOPTIONS` in your `install-jetty-service.bat` script:
 
-    ;-Dephox.config.file="C:\config\file\location\application.conf"
+```
+;-Dephox.config.file="C:\config\file\location\application.conf"
```

Comment:
^ that command doesn't look right

Reply:
apparently these are correct, although I need to review this content as some point